src/biguint.rs

```diff
@@ -1749,6 +1795,25 @@ impl BigUint {
         }
         acc
     }
+
+    /// Returns the truncated principal square root of `self` --
+    /// see [Roots::sqrt](Roots::sqrt).
+    // struct.BigInt.html#trait.Roots
```

Comment:
These links don't look right -- did you forget to complete this?

Reply:
Eh, forgot to remove.